twint/src/main/java/com/adyen/checkout/twint/Twint.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 17/11/2023.
+ */
+
+package com.adyen.checkout.twint
+
+import androidx.activity.ComponentActivity
+import androidx.activity.result.ActivityResultLauncher
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import ch.twint.payment.sdk.Twint
+import ch.twint.payment.sdk.TwintPayResult
+import com.adyen.checkout.components.core.action.SdkAction
+import com.adyen.checkout.components.core.action.TwintSdkData
+import com.adyen.checkout.core.exception.CheckoutException
+import com.adyen.checkout.twint.Twint.initialize
+
+/**
+ * Object used to manage the Twint SDK. Call [initialize] in [ComponentActivity.onCreate] to make sure [SdkAction]s
+ * with [TwintSdkData] can be handled.
+ */
+object Twint {
+
+    private var twintObject: Twint? = null
+
+    private var onResultListener: ((TwintPayResult) -> Unit)? = null
+
+    /**
+     * Initializes the [Twint] object. This method **must** be called in [ComponentActivity.onCreate] or before, because
```

Comment:
Minor: I understand the reason behind mentioning `ComponentActivity` but I'm wondering if saying "your activity's onCreate" could be clearer, WDYT?

Reply:
Yeah it sounds more clear and it should already be clear that a `ComponentActivity` should be used